docs/mynewfile.md

```diff
@@ -0,0 +1,3 @@
+### This is my new file!! 
+
+##### OMG, this is too easy :)
```

Comment:
I should add my favorite color the next time or right now if the PR is still open